lambdas/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.11-alpine
+FROM python:3.11
```

Comment:
Could `python:3.11-slim` be a better option. its compatible to have polars library as its larger than alpine yet smaller than `python:3.11`?

Reply:
One of the articles we came across showed a user who was using python:3.11-slim and they were encountering the same issue as us when trying to install polars, hence why we modified it to use the standard image as supported by the polars team.